assets/js/modules/analytics/dashboard/dashboard-widget-top-earning-pages-small.js

```diff
@@ -107,23 +107,17 @@ class AdSenseDashboardWidgetTopPagesTableSmall extends Component {
 	}
 }
 
-const isDataZero = () => {
-	return false;
-};
-
 /**
  * Check error data response.
  *
- * @param {Object} data Response data.
+ * @param {Object} data Response error data.
  *
  * @return {(HTMLElement|null)} Returns HTML element markup with error message if it exists.
  */
 const getDataError = ( data ) => {
-	if ( data && data.error_data ) {
-		const errors = Object.values( data.error_data );
-
+	if ( data.code && data.message && data.data && data.data.status ) {
 		// Specifically looking for string "badRequest"
-		if ( errors[ 0 ] && 'badRequest' === errors[ 0 ].reason ) {
+		if ( data.data.reason && 'badRequest' === data.data.reason ) {
```

Comment:
Same as above, this could be a bit cleaner and simplified.

Reply:
See above, I'll only omit the one unnecessary check.